thepracticaldev/dev.to
# [ModCenter] Update view heading and highlight chosen tag, on tag selection

## What type of PR is this? (check all applicable)


## Description
Updates the List heading after choosing a tag; also highlights the chosen tag

## QA Instructions, Screenshots, Recordings
1. Fire up your local environment
2. Navigate to `localhost:port/mod/`
3. Choose any of the tags under `Inbox`
4. Ensure that the list heading changes to the tag you chose
5. Ensure that the tag you chose is highlighted (and none others are)
6. Ensure that the list of articles returned ALL have the tag you chose

![update-tag-view-heading-PR](https://user-images.githubusercontent.com/32520970/84547185-062d5d00-acd1-11ea-8af8-7cfdd3220327.gif)

## Added tests?

- [X] no, not yet

## Added to documentation?


## [optional] What gif best describes this PR or how it makes you feel?

![Minions](https://media.giphy.com/media/lqFY9hBTLX7os/giphy.gif)
